build-system/git.js

```diff
@@ -84,29 +86,17 @@ exports.gitDiffStatMaster = function() {
 
 /**
  * Returns a detailed log of commits included in a PR check, starting with (and
- * including) the branch point off of master. Limited to at most 100 commits to
- * keep the output sane.
+ * including) the branch point off of master. Limited to commits in the past
+ * 30 days to keep the output sane.
  *
  * @return {string}
  */
 exports.gitDiffCommitLog = function() {
-  const branchPoint = exports.gitMergeBaseMaster();
-  let commitLog = getStdout(`git -c color.ui=always log --graph \
+  const branchCreationPoint = exports.gitBranchCreationPoint();
+  const commitLog = getStdout(`git -c color.ui=always log --graph \
 --pretty=format:"%C(red)%h%C(reset) %C(bold cyan)%an%C(reset) \
 -%C(yellow)%d%C(reset) %C(reset)%s%C(reset) %C(green)(%cr)%C(reset)" \
---abbrev-commit ${branchPoint}^...HEAD \
---max-count=${commitLogMaxCount}`).trim();
-  if (commitLog.split('\n').length >= commitLogMaxCount) {
-    commitLog += `\n${colors.yellow('WARNING:')} Commit log is longer than \
-${colors.cyan(commitLogMaxCount)} commits. \
-Branch ${colors.cyan(exports.gitBranchName())} may not have been forked from \
-${colors.cyan('master')}.`;
-    commitLog += `\n${colors.yellow('WARNING:')} See \
-${colors.cyan(
-  'https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-quick.md'
-)} \
-for how to fix this.`;
-  }
+--abbrev-commit ${branchCreationPoint}^...HEAD --since "30 days ago"`).trim();
```

Comment:
Why change the limit to 30 days instead of 100 commits?

Reply:
The commit log is printed because Travis creates a merge commit (with new commits from `master`) which sometimes causes checks or tests to fail unexpectedly. Since PRs can be worked on for multiple days or even weeks, it's useful to know what commits have landed since the PR branch was forked. 

Occasionally, 100 commits isn't sufficient because of the age / size of a PR. See [this](https://travis-ci.org/ampproject/amphtml/jobs/533529829#L456) example. I picked 30 days because it seemed like a reasonable time period for working on a PR. Short / recent PRs will continue to show short commit logs.